[2021-09-27 14:08:19,222] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.delete_entry 2021-09-27T14:06:40.788506+00:00 [queued]>
[2021-09-27 14:08:19,237] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.delete_entry 2021-09-27T14:06:40.788506+00:00 [queued]>
[2021-09-27 14:08:19,238] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:08:19,239] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 14:08:19,240] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:08:19,260] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): delete_entry> on 2021-09-27T14:06:40.788506+00:00
[2021-09-27 14:08:19,264] {standard_task_runner.py:52} INFO - Started process 2337 to run task
[2021-09-27 14:08:19,270] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'delete_entry', '2021-09-27T14:06:40.788506+00:00', '--job-id', '39', '--pool', 'default_pool', '--raw', '--subdir', '/home/***/.local/lib/python3.6/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmp_dlw3ua_', '--error-file', '/tmp/tmpluzwe4_3']
[2021-09-27 14:08:19,277] {standard_task_runner.py:77} INFO - Job 39: Subtask delete_entry
[2021-09-27 14:08:19,330] {logging_mixin.py:109} INFO - Running <TaskInstance: example_complex.delete_entry 2021-09-27T14:06:40.788506+00:00 [running]> on host 77d12c45e78e
[2021-09-27 14:08:19,395] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_complex
AIRFLOW_CTX_TASK_ID=delete_entry
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T14:06:40.788506+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-27T14:06:40.788506+00:00
[2021-09-27 14:08:19,397] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-27 14:08:19,399] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo delete_entry']
[2021-09-27 14:08:19,407] {subprocess.py:74} INFO - Output:
[2021-09-27 14:08:19,409] {subprocess.py:78} INFO - delete_entry
[2021-09-27 14:08:19,410] {subprocess.py:82} INFO - Command exited with return code 0
[2021-09-27 14:08:19,454] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=delete_entry, execution_date=20210927T140640, start_date=20210927T140819, end_date=20210927T140819
[2021-09-27 14:08:19,490] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 14:08:19,524] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
